src/pages/new/index.js

```diff
@@ -63,35 +62,28 @@ export default () => {
           title: "Volunteer",
           description:
             "Volunteers make our work possible. There are several ways to support our mission. Learn more and get involved.",
+          button: { text: "Become a Volunteer", internalLink: "/foo" },
         }}
         paragraph2={{
           title: "Partnerships",
           description:
             "We work with companies, nonprofits, and local governments to empower the community. Reach out to us.",
+          button: { text: "Work With Us", internalLink: "/bar" },
```

Comment:
Similar to @davidagustin's comments, this button should open up the volunteer modal like before.